data/json/itemgroups/locations.json

```diff
@@ -734,23 +734,23 @@
       { "item": "oil_lamp", "prob": 20, "charges-min": 0, "charges-max": 3 },
       [ "steam_triple_small", 1 ],
       [ "steam_triple_medium", 1 ],
-      [ "waffleiron", 20 ],
-      [ "spring", 45 ],
+      [ "waffleiron", 60 ],
+      [ "spring", 150 ],
       { "item": "rebar_rail", "prob": 45, "charges-min": 1, "charges-max": 50 },
-      [ "pastaextruder", 25 ],
-      { "item": "nail", "prob": 65, "charges-min": 1, "charges-max": 15 },
-      [ "well_pump", 15 ],
+      [ "pastaextruder", 75 ],
+      { "item": "nail", "prob": 300, "charges-min": 1, "charges-max": 15 },
+      [ "well_pump", 50 ],
       [ "manhole_cover", 20 ],
-      [ "hairpin", 40 ],
-      [ "pan", 30 ],
+      [ "hairpin", 120 ],
+      [ "pan", 90 ],
```

Comment:
I've added a ton of kitchens pots and pans that could be dumped into this group too

Reply:
I'll look around what can I throw in.